BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -174,4 +177,52 @@ public void EditProfileUpdate_CalledUsingInvalidModel_Returns404Error()
         // Assert   
         result.Should().BeOfType<StatusCodeResult>().Which.StatusCode.Should().Be(404);
     }
+
+    [Fact]
+    public void ViewProperties_CalledByAdmin_CanReturnOtherUsersPropertyList()
+    {
+        // Arrange
+        var adminUser = CreateAdminUser();
+        adminUser.Id = 1;
+        MakeUserPrincipalInController(adminUser, UnderTest);
+        A.CallTo(() => PropertyService.GetPropertiesByLandlord(2))
+            .Returns(A.CollectionOfFake<PropertyDbModel>(10).ToList());
+
+        // Act
+        var result = UnderTest.ViewProperties(2) as ViewResult;
+
+        // Assert
+        result.Should().BeOfType<ViewResult>();
+        result.Should().NotBeNull();
+        if (result != null)
+        {
+            result!.Model.Should().BeOfType<PropertiesDashboardViewModel>();
+            result.Model.As<PropertiesDashboardViewModel>().Properties.Count.Should()
+                .Be(10);
+        }

```

Comment:
nit don't need the if - the test will already have failed above
```suggestion
        result!.Model.Should().BeOfType<PropertiesDashboardViewModel>();
        result.Model.As<PropertiesDashboardViewModel>().Properties.Count.Should()
            .Be(10);
```